cypress/support/config/settings.js

```diff
@@ -2231,7 +2231,7 @@ module.exports = () => ({
             enabled: true,
           },
         },
-        smoke: false,
+        smoke: true,
```

Comment:
This enables smoke tests for hindi On Demand TV on local (which will fail, as the tests are expecting an iFrame)

Reply:
Marking this as resolved on account of the tests having been updated to
assert for existence of the MediaLoader component and the placeholder image.